src/routes/auth/controllers/githubCallback.ts

```diff
@@ -0,0 +1,34 @@
+import { Context, Next } from 'koa';
+import fetch from 'node-fetch';
+
+const clientId = process.env.GITHUB_OAUTH_CLIENT_ID;
+const clientSecret = process.env.GITHUB_OAUTH_CLIENT_SECRET;
```

Comment:
```suggestion
const CLIENT_ID = process.env.GITHUB_OAUTH_CLIENT_ID;
const CLIENT_SECRET = process.env.GITHUB_OAUTH_CLIENT_SECRET;
```

변수 선언을 대문자로 바꾸는거 어떻게 생각하세요??